Tests/SwiftCurrent_SwiftUITests/SwiftCurrent_SwiftUITests.swift

```diff
@@ -703,9 +678,27 @@ final class SwiftCurrent_SwiftUIConsumerTests: XCTestCase {
         let workflowView = WorkflowLauncher(isLaunched: .constant(true))
             .thenProceed(with: WorkflowItem(FR1.self))
 
-        let explorer = unsafeBitCast(workflowView, to: ModifiedWorkflowViewExplorer<Never, Never, FR1>.self)
+        typealias WorkflowViewContent = State<WorkflowItem<FR1, Never, FR1>>
+        let content = try XCTUnwrap(Mirror(reflecting: workflowView).descendant("_content") as? WorkflowViewContent)
+
+        // Note: Only add to these exceptions if you are *certain* the property should not be @State. Err on the side of the property being @State
+        let exceptions = ["_model", "_launcher", "_location", "_value"]
```

Comment:
Aren't these `@State`?  Or is the state `model` and `_model` is the non-state thing?

Reply:
No, `_model` is an `EnvironmentObject`, not @State. `_location` and `_value` are SwiftUI things that are exposed via reflection.